.gitlab-ci.yml

```diff
@@ -61,19 +64,23 @@ nix-linux:
     - public/
     expire_in: 1 week
 
-nix-shell-linux:
-  tags:
-    - linux
-  script:
-    - nix-shell --run "echo Done with shell test"
-
 # 2019 Feb 11 - This is bottlenecking CI quite severely.
 # stack:
 #   tags:
 #     - macos
 #   script:
 #     - nix-shell --pure -p stack libiconv zlib ncurses gmp git perl darwin.security_tool --run "stack test --fast"
 
+slow-tests:
+  stage: test
+  only:
+    - master
+  tags:
+    - linux
+  script:
+    - echo This is a master slow test run.
+    - result/ghc/chainweb/bin/slow-tests
```

Comment:
slow-tests does also contain ssl-certificate related tests that caused issues in nix-sandboxes in the past. Are we sure they work here? Otherwise, one can pass `--pattern=/consensus/` to slow-tests to only run the consensus tests.

Reply:
This won't be a problem because it's being run directly, not in a nix sandbox.